### PR TITLE
Create Simple RecyclerView

### PR DESCRIPTION
**Pull request by natalliarad**
Add layout files

Add hard code content

Add ArticlesRecyclerAdapter, ArticlesViewHolder classes and 2 interfaces: ArticleItemView, FragmentCoordinator

Add 2 fragments: FragmentArticleRecyclerView, FragmentArticleDetail. Edit MaimActivity

Change build.gradle file

Add layout files

Add hard code content

Add ArticlesRecyclerAdapter, ArticlesViewHolder classes and 2 interfaces: ArticleItemView, FragmentCoordinator

Add 2 fragments: FragmentArticleRecyclerView, FragmentArticleDetail. Edit MaimActivity

### CODE REVIEW

File: app/src/main/java/com/example/natallia_radaman/simplerecyclerview/ArticlesRecyclerAdapter.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.natallia_radaman.simplerecyclerview.HardCodeContent.HardCodeArticle;
+
+import java.util.List;
+
+public class ArticlesRecyclerAdapter extends RecyclerView.Adapter<ArticlesViewHolder> {
+
+    private List<HardCodeArticle> articleList;
```
Comment (rooton) [inline app/src/main/java/com/example/natallia_radaman/simplerecyclerview/ArticlesRecyclerAdapter.java]: Can we add `final` modifier for articleList, context, callbackActivity?

File: app/src/main/java/com/example/natallia_radaman/simplerecyclerview/ArticlesRecyclerAdapter.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    private Context context;
+    private FragmentCoordinator callbackActivity;
+
+    public ArticlesRecyclerAdapter(List<HardCodeArticle> articleList, Context context) {
```
Comment (rooton) [inline app/src/main/java/com/example/natallia_radaman/simplerecyclerview/ArticlesRecyclerAdapter.java]: Will articleList or context change later in constructor body? If not, then why not add `final` modifier? How do I know is articleList or context  maybe nullable or not?

File: app/src/main/java/com/example/natallia_radaman/simplerecyclerview/ArticleItemView.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.example.natallia_radaman.simplerecyclerview;
```
Comment (PavelDolbik) [inline app/src/main/java/com/example/natallia_radaman/simplerecyclerview/ArticleItemView.java]: App crashed !!!
` java.lang.NullPointerException: Attempt to invoke virtual method 'int android.os.Bundle.getInt(java.lang.String)' on a null object reference`

I would like you suggest rewrite this app. (only one activity + adapter)

Comment (natalliarad) [reply app/src/main/java/com/example/natallia_radaman/simplerecyclerview/ArticleItemView.java]: Ok. Now I'm doing it.

Comment (natalliarad) [reply app/src/main/java/com/example/natallia_radaman/simplerecyclerview/ArticleItemView.java]: Done.

Comment (natalliarad) [reply app/src/main/java/com/example/natallia_radaman/simplerecyclerview/ArticleItemView.java]: Pasha, I did a recyclerView with two classes. 
Look repository one more time, please.

Thanks in advance.

**Pull request merged**